worker/src/test/scala/org/apache/celeborn/service/deploy/MiniClusterFeature.scala

```diff
@@ -202,16 +203,16 @@ trait MiniClusterFeature extends Logging {
                 logError(s"cannot start worker $i, reached to max retrying", ex)
                 throw ex
               } else {
-                Thread.sleep(math.pow(2000, workerStartRetry).toInt)
+                TimeUnit.SECONDS.sleep(Math.pow(2, workerStartRetry).toLong)
               }
           }
         }
       })
-      workerThread.setName(s"worker ${i} starter thread")
+      workerThread.setName(s"worker $i starter thread")
       workerThread
     }
     threads.foreach(_.start())
-    Thread.sleep(5000)
+    Thread.sleep(15000)
```

Comment:
QQ: Why did we bump this from 5 -> 15 ?

Reply:
Because `assert` is used here, it does not have a chance to retry before this PR.

https://github.com/apache/celeborn/blob/0ee3c3a4bdef3b97fca6be04f03ac36c2e12e1a6/worker/src/test/scala/org/apache/celeborn/service/deploy/MiniClusterFeature.scala#L228-L232